### PR TITLE
Fix zynthian/zynthian-issue-tracking#817: Fix preset restore after preload,

### PR DESCRIPTION
**Pull request by jofemodo**
so it's always done after exiting preset/bank screen.

### CODE REVIEW

File: zynthian_gui.py [hunk 1]
```diff
@@ -721,6 +721,9 @@ def show_screen(self, screen=None, hmode=SCREEN_HMODE_ADD):
 			else:
 				return
 
+		if screen not in ("bank", "preset", "option"):
```
Comment (riban-bw) [inline zynthian_gui.py:724]: This function gets called a lot - one almost every screen navigation.

Comment (riban-bw) [reply zynthian_gui.py:724]: If pre-selection is toggled as favourite then this still fails, i.e. preset is restored.

**Review by riban-bw — COMMENTED** [review]
I don't think this is sufficiently urgent to warrant a hot fix. It only affects Z2. Can it wait until a point release?

**Comment by jofemodo** [comment]
It affects V4 too. For instance, from preset preloading, if you press bold-back to open main menu, engine-selected preset is not restored either, etc. 
Also, i think this could solve https://github.com/zynthian/zynthian-issue-tracking/issues/764. I "kind of reproduced" #764 when trying to solve this, and after the fix, i didn't see #764 again.
We should merge to testing.

**Pull request merged**